.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+yarn.lock
```

Comment:
@sindresorhus thoughts on ignoring lockfiles? If :+1: then we also need to ignore `package-lock.json`.

Should probably also include `yarn-error.log` and `npm-debug.log` regardless.

Reply:
I added the rest, if we need to remove it let me know. I looked at chalk and yarn.lock was ignored so i figured it was the same here.

If i'm wrong i will remove it :)